### ISSUE TITLE
Cannot connect Electrum to EPS - Error with localhost:127.0.0.1:50002

### ISSUE THREAD

**Issue by qherring** [chris-belcher/electrum-personal-server#292]
When I run electrum after staring up the node, and EPS i simply cannot get electrum to connect to EPS over 127.0.0.1:50002

I am running Bitcoin Core version v24.0.1, Electrum v4.4.2,  and EPS v0.2.4
This is being run through tor

EPS connects to Bitcoin Core without any issues.  Bitcoin Core is fully up to date and synced to current block:
```
INFO:2023-05-09 10:58:37,711: Found 47749 mempool entries. Synchronized mempool in 11.534810543060303sec
INFO:2023-05-09 10:58:37,712: Listening for Electrum Wallet on ('127.0.0.1', 50002) 
```

When I run electrum -v I am getting an error about certificate expired:

```
0.53 | I | i/interface.[127.0.0.1:50002] | certificate has expired: Certificate (unknown) has expired (at 2023-03-12 08:07:58 UTC).
  0.54 | I | storage.WalletStorage | wallet path /home/username/.electrum/wallets/public ledger
  0.65 | I | i/interface.[127.0.0.1:50002] | succeeded in getting cert
  0.86 | W | i/interface.[127.0.0.1:50002] | Cannot connect to main server due to SSL error (maybe cert changed compared to "/home/username/.electrum/certs/127.0.0.1"). Exc: ConnectError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:997)'))
  0.86 | I | n/network | couldn't launch iface 127.0.0.1:50002:s -- CancelledError()
```

So then if I run electrum -v --oneserver --server localhost:50002:s.    I get the following:

```
 0.44 | I | n/network | starting network
  0.44 | W | n/network | failed to parse server-string ('localhost:50002:s.'); falling back to localhost:1:s.
  0.44 | I | n/network | setting proxy {'mode': 'socks5', 'host': '127.0.0.1', 'port': '9050', 'user': '', 'password': ''}
  0.44 | I | storage.WalletStorage | wallet path /home/username/.electrum/wallets/public ledger
  0.44 | I | n/network | Proxy is TOR
  0.44 | I | n/network | connecting to localhost:1:s as new interface
  0.44 | I | i/interface.[localhost:1] | looks like localhost: not using proxy for this server
  0.44 | I | n/network | starting taskgroup (0x7f19c899c3a0).
  0.44 | D | util.profiler | Daemon._load_wallet 0.0013 sec
  0.52 | I | i/interface.[localhost:1] | disconnecting due to: ErrorGettingSSLCertFromServer(ConnectError(ConnectionRefusedError(111, "Connect call failed ('127.0.0.1', 1)")))
  0.52 | I | n/network | couldn't launch iface localhost:1:s -- CancelledError()
```

What am I doing wrong here?

**Comment by qherring** [comment]
I tried to create my own SSL cert following this guide: https://github.com/spesmilo/electrum-server/blob/ce1b11d7f5f7a70a3b6cc7ec1d3e552436e54ffe/HOWTO.md#step-8-create-a-self-signed-ssl-cert

then I updated the cert files for EPS (cert.crt and cert.key) with the output of the previous step.

then I deleted the cert file from electrum (127.0.0.1)

When trying electrum -v again, electrum just recreates a cert files 127.0.0.1 with the old certificate.  Where is it updating from?  and how can I get it to use the current one?
